bigtable/integration_test.go

```diff
@@ -2237,7 +2237,7 @@ func TestIntegration_InstanceUpdate(t *testing.T) {
 func TestIntegration_AdminBackup(t *testing.T) {
 	testEnv, err := NewIntegrationEnv()
 	if err != nil {
-		t.Fatalf("IntegrationEnv: %v", err)
+		t.Errorf("IntegrationEnv: %v", err)
```

Comment:
If `NewIntegrationEnv` fails, does it make sense to continue the test? `Fatalf` makes sense to me.